plugins/MauticFocusBundle/Entity/Stat.php

```diff
@@ -38,6 +38,9 @@ class Stat
      */
     private $dateAdded;
 
+    /**
+     * @var Lead
```

Comment:
```suggestion
     * @var ?Lead
```
When you create a fresh entity then this is null. 
```php
$stat = new Stat();
$stat->getLead(); // returns null, not Lead
```

Reply:
I changed this.